docs/content/en/docs/getting-started/optional/irsa.md

```diff
@@ -45,66 +45,65 @@ You must use a single OIDC provider per EKS Anywhere cluster, which is the best
+
+	1. Select _Web identity_ as the trusted entity.
+
+	1. In the _Web identity_ section:
+
+		* If your _Identity provider_ is not auto selected, select it.
+		* Select `sts.amazonaws.com` as the _Audience_.
+
+	1. Click _Next_.
+
+	1. Configure your desired _Permissions poilicies_.
+
+	1. Below is a sample trust policy of IAM role for your pods. Replace `ACCOUNT_ID`, `ISSUER_HOSTPATH`, `NAMESPACE` and `SERVICE_ACCOUNT`.
+        _Example: Scoped to a service account_
+        ```json
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Principal": {
+                        "Federated": "arn:aws:iam::ACCOUNT_ID:oidc-provider/ISSUER_HOSTPATH"
+                    },
+                    "Action": "sts:AssumeRoleWithWebIdentity",
+                    "Condition": {
+                        "StringEquals": {
+                            "ISSUER_HOSTPATH:sub": "system:serviceaccount:NAMESPACE:SERVICE_ACCOUNT"
+                        },
+                    }
+                }
+            ]
+        }
        ```
-       
-	1. After the role is created, note down the name of this IAM Role as `OIDC_IAM_ROLE`. 
 
-	1. Once the cluster is created, you can create service accounts and grant them this role by editing the trust relationship of this role. You can use `StringLike` condition to add required service accounts, as mentioned in the above sample. Please also refer to section [Configure the trust relationship for the OIDC provider's IAM Role](#configure-the-trust-relationship-for-the-oidc-providers-iam-role).
+	1. Create the IAM Role and make a note of the _Role name_.
+
+	1. After the cluster is created you can grant service accounts access to the role by modifying the trust relationship. See the [How to use trust policies with IAM Roles](https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/) for more information on trust policies. Refer to [Configure the trust relationship for the OIDC provider's IAM Role](#configure-the-trust-relationship-for-the-oidc-providers-iam-role) for a working example.
 
 ### Create the EKS Anywhere cluster
 
-1. When creating the EKS Anywhere cluster, you need to configure the kube-apiserver's `service-account-issuer` flag so it can issue and mount projected service account tokens in pods. For this, use the value obtained in the first section for `$ISSUER_HOSTPATH` as the `service-account-issuer`. Configure the kube-apiserver by setting this value through the EKS Anywhere cluster spec as follows:
+1. When creating the EKS Anywhere cluster, you need to configure the kube-apiserver's `service-account-issuer` flag so it can issue and mount projected service account tokens in pods. For this, use the value obtained in the first section for `$ISSUER_HOSTPATH` as the `service-account-issuer`. Configure the kube-apiserver by setting this value through the EKS Anywhere cluster spec:
```

Comment:
```suggestion
When creating the EKS Anywhere cluster, you need to configure the kube-apiserver's `service-account-issuer` flag so it can issue and mount projected service account tokens in pods. For this, use the value obtained in the first section for `$ISSUER_HOSTPATH` as the `service-account-issuer`. Configure the kube-apiserver by setting this value through the EKS Anywhere cluster spec:
```